sysid-application/src/main/native/include/sysid/view/AnalyzerPlot.h

```diff
@@ -112,6 +112,8 @@ class AnalyzerPlot {
    */
   double* GetSimRSquared();
 
+  float m_pointSize = 1.25;
```

Comment:
We generally put variables first, then functions in a given access specifier block.